docs/status.md

```diff
@@ -1,4 +1,4 @@
-## About
+# StableHLO progress
```

Comment:
How about "StableHLO Status"?

Reply:
Was this made by markdownlint? Or was this made separately because it is more readable?

Reply:
SGTM, done. I made this change because "About" was an H2 heading, so this document didn't actually have a proper title (markdownlint requires each file have an H1 heading). I chose "progress" based on the sentence in here that says, "The progress of a StableHLO op... is tracked using one of the following labels," and also because "Status" is already used in an H2 heading, but I'm fine with either.